core/client/helpers.js

```diff
@@ -303,19 +303,22 @@ const vectorToTextDirection = vector => {
   return undefined;
 };
 
+const isMobile = () => window.innerWidth < 480;
```

Comment:
Isn't it better to rely on "window.matchMedia('(pointer: coarse)').matches"? The raw screen resolution scares me a bit, especially with Retina screens it's a bit misleading